getting-started/deployment_guide.md

```diff
@@ -97,3 +99,34 @@ karmor log
 3. [Minikube](../contribution/minikube#minikube-installation)
 4. [Self-managed K8s](../contribution/self-managed-k8s)
 5. [Amazon Elastic Kubernetes Service (EKS)](../deployments/EKS#prerequisite-for-the-deployment)
+6. Setting Environment for KubeArmor
+
```

Comment:
We are essentially replacing the Minikube bullet point with the collapsible section.. Can you take care of that? Thank You. Eventually all the references to seperate documentation will be collapsed into here.

Reply:
So the minikube section needs to be collapsible and the Minikube Installation should be in that section, Am I right ? @daemon1024

Reply:
Yes

Reply:
Hey @daemon1024 should I replace all the bullet points with collapsible(It does not looks good with minikube is collapsible and other are bullet points), What's your take?

Reply:
Eventually we will be replacing all of it with collapsible section. We still want bullet points, you can refer to my comment at https://github.com/kubearmor/KubeArmor/pull/698#discussion_r878702277